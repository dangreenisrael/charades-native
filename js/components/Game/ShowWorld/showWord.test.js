import 'react-native';
import renderer from 'react-test-renderer';
import React from 'react';
import {ShowWord} from '.';

test('renders correctly', () => {
  const tree = renderer.create(<ShowWord />).toJSON();
  expect(tree).toMatchSnapshot();
});
