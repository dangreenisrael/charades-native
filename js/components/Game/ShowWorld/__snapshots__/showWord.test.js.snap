// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
  style={
    Object {
      "color": "#000",
      "fontFamily": "System",
      "fontSize": 40,
      "lineHeight": 50,
      "marginBottom": 100,
      "textAlign": "center",
    }
  }
  virtual={undefined}
>
  Tap For a Word
</Text>
`;
